### PR TITLE
bsunanda:Phase2-ft15 Bug fix of FastTimer Reco Geometry (backporting from 9_0_X)

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalGeometry/test/python/testFastTimeGeometry_cfg.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    debugModules = cms.untracked.vstring('*'),
+    cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('DEBUG'),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        HGCalGeom = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        )
+    ),
```
Comment (ianna) [inline Geometry/HGCalGeometry/test/python/testFastTimeGeometry_cfg.py]: @bsunanda - the same comment - this comma is a typo?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
not needed in 81x. (phase2 dev will evolve in 90x)

**Pull request closed without merging**